src/service/faas.service.ts

```diff
@@ -276,12 +313,12 @@ Please make sure the function with the name ${name} was pushed to the LivePerson
       const urlPart = `/lambdas${uuid ? `/${uuid}` : ''}`;
       await this.doFetch({ urlPart, method, body });
     } catch (error) {
-      if (error.error?.errorCode?.includes('contract-error')) {
+      if (error.errorCode?.includes('contract-error')) {
         throw new Error(
           `Push Error: The code of function '${body.name}' you are trying to push is not a valid lambda.`,
         );
       }
-      throw error;
+      throw new Error(error.errorMsg);
```

Comment:
Should we provide here a fallback if error.errorMsg is undefined ? Not sure if that can happen